htlcswitch/link.go

```diff
@@ -887,20 +908,34 @@ func (l *channelLink) htlcManager() {
 			// We have lost state and cannot safely force close the
 			// channel. Fail the channel and wait for the remote to
 			// hopefully force close it. The remote has sent us its
-			// latest unrevoked commitment point, that we stored in
-			// the database, that we can use to retrieve the funds
-			// when the remote closes the channel.
+			// latest unrevoked commitment point, and we'll store
+			// it in the database, such that we can attempt to
+			// recover the funds if the remote force closes the
+			// channel.
 			// TODO(halseth): mark this, such that we prevent
 			// channel from being force closed by the user or
 			// contractcourt etc.
-			case err == lnwallet.ErrCommitSyncLocalDataLoss:
+			case localDataLoss:
```

Comment:
I think the TODO above can be removed since `LightningChannel.ForceClose` will check whether it has a `ChanStatusLocalDataLoss` status before doing so.

Reply:
Done!